lightning/src/chain/chaininterface.rs

```diff
@@ -19,8 +19,17 @@ use bitcoin::blockdata::transaction::Transaction;
 
 /// An interface to send a transaction to the Bitcoin network.
 pub trait BroadcasterInterface {
-	/// Sends a transaction out to (hopefully) be mined.
-	fn broadcast_transaction(&self, tx: &Transaction);
+	/// Sends a list of transactions out to (hopefully) be mined.
+	/// This only needs to handle the actual broadcasting of transactions, LDK will automatically
+	/// rebroadcast translations that haven't made it into a block.
+	///
+	/// In some cases LDK may attempt to broadcast a transaction which double-spends another
+	/// and this isn't a bug and can be safely ignored.
```

Comment:
I’m not sure if this comment makes sense in the light of local network mempool partition. Your mempool yielding a conflict could have been provoked by another party or even an issue in the deployment of new mempool policy rules on the network. This could be investigated by the node operator, and even manual actions take like re-broadcasting your transactions to another Bitcoin Core node.

That let me think maybe we should have a buffer of transactions and a method to “fetch” all our non-confirmed transactions in byte format. This can be interesting for the VLS recovery mode flow.